astropy/table/table.py

```diff
@@ -2487,13 +2497,22 @@ def sort(self, keys=None):
 
         Sorting according to standard sorting rules, first 'name' then 'firstname'::
 
-            >>> t.sort(['name','firstname'])
+            >>> t.sort(['name', 'firstname'])
             >>> print(t)
             firstname   name  tel
             --------- ------- ---
                  John Jackson  18
                    Jo  Miller  15
                   Max  Miller  12
+
+        Sorting according to standard sorting rules, first 'firstname' then 'tel', in reverse order::
+            >>> t.sort(['firstname', 'tel'], reverse=True)
```

Comment:
Could you add a blank line between the text and the code?

Reply:
Okay done @astrofrog

Reply:
Oops ! Did the tests fail because of the merge conflicts or did I mess something up in rebasing?